metal3-templater.go

```diff
@@ -64,6 +70,14 @@ func main() {
 	templateData.ClusterIronicURL = fmt.Sprintf("http://%s", net.JoinHostPort(ip, "6385"))
 	templateData.ClusterInspectorURL = fmt.Sprintf("http://%s", net.JoinHostPort(ip, "5050"))
 
+	// http_basic credentials
+	if numArgs == 11 {
+		templateData.IronicUser = fmt.Sprintf("%s", os.Args[7])
```

Comment:
I don't think the `Sprintf()` calls are needed here, are they? Can't we just assign the array element to the field of the template struct?

Reply:
Done